filament/backend/src/vulkan/VulkanTexture.cpp

```diff
@@ -387,8 +390,10 @@ void VulkanTexture::setPrimaryRange(uint32_t minMiplevel, uint32_t maxMiplevel)
 }
 
 VkImageView VulkanTexture::getAttachmentView(VkImageSubresourceRange range) {
```

Comment:
We should probably add getMultiviewAttachmentView instead of overloading this one.